fields/types/date/test/server.js

```diff
@@ -8,36 +9,106 @@ exports.initList = function (List) {
 };
 
 exports.testFieldType = function (List) {
-	var testItem = new List.model();
-
 	it('should parse date input and return a moment object', function () {
 		var m = List.fields.date.parse('2016-02-27');
 		demand(m.format('YYYY-MM-DD')).equal('2016-02-27');
 	});
 
 	it('should format the date value using moment', function () {
+		var testItem = new List.model();
 		testItem.date = new Date(2013, 11, 4);
 		demand(testItem._.date.format()).equal('4th Dec 2013');
 		demand(testItem._.date.format('YYYYMMDD')).equal('20131204');
 	});
 
 	it('should return a moment object set to the field value', function () {
+		var testItem = new List.model();
 		testItem.date = new Date(2013, 11, 4);
 		demand(testItem._.date.moment()._isAMomentObject);
 		demand(testItem._.date.moment().format('YYYYMMDD')).equal('20131204');
 	});
 
-	it('should validate input present', function (done) {
-		List.fields.date.validateRequiredInput(this, { date: 'a' }, function (result) {
-			demand(result).be(true);
-			done();
-		});
+	it('should use the common text input validator', function () {
+		demand(List.fields.date.validateRequiredInput === TextType.prototype.validateRequiredInput);
 	});
 
-	it('should invalidate input not present', function (done) {
-		List.fields.date.validateRequiredInput(this, { date: '' }, function (result) {
-			demand(result).be(false);
-			done();
+	describe('validateInput', function () {
+		it('should validate date strings', function (done) {
+			List.fields.date.validateInput({ date: '2015-01-01' }, function (result) {
+				demand(result).be(true);
+				done();
+			});
+		});
+
+		it('should invalidate unformatted dates', function (done) {
+			List.fields.date.validateInput({ date: new Date(2015, 1, 1) }, function (result) {
```

Comment:
The reason this test fails is because the date is a string in the wrong format (something like "Thu 28th March 2016 blabla") which moment doesn't want because we have a different default format string. Is this intended behaviour?